Change so that default presentation delay for DASH is configurable #1234

bwallbergBonnier opened the pull request:
The DEFAULT_SUGGESTED_PRESENTATION_DELAY_ in DashParser is useful to have configurable
when the stream provider isn't able to add `suggestedPresentationDelay` in the manifest.

Fixes #1234

joeyparrish commented on externs/shaka/player.js:
```diff
@@ -508,6 +509,9 @@ shakaExtern.DrmConfiguration;
  *   false if not provided.
+ * @property {number} defaultPresentationDelay
+ *   A default presentationDelay if suggestedPresentationDelay is missing
+ *   in the MPEG DASH manifest, has to be biggar than minBufferTime * 1.5.
```
typo: "biggar" should be "bigger"

joeyparrish commented:
It looks like jsdoc doesn't like your trailing comma in the doc string.  I'll file a bug to make jsdoc more flexible, as trailing commas are recommended in other contexts.

bwallbergBonnier commented:
@joeyparrish 
I've amended the commit and removed the trailing comma and fixed the spelling mistake :)
Is there a command I should've run to see the jsdoc error? I ran `./build/all.py` and `./build/test.py` and didn't get that error 🤔

joeyparrish commented:
Yes `./build/docs.py` will build the documentation.  It is a little silly and counterintuitive that `all.py` doesn't include `docs.py`, but we reasoned that someone who wants to rebuild the code all the time during development won't want to rebuild the docs all the time.  I'm honestly unsure what the best change would be to improve that.

joeyparrish commented:
Okay, everything looks good!  Thanks!

joeyparrish commented:
Cherry-picked for v2.3.2.  Just barely missed the cut-off for v2.3.1.  Thanks again for the contribution!
